docs/source/guides/developing_bits.rst

```diff
@@ -0,0 +1,48 @@
+Developing BITS
```

Comment:
This is the page title.  Needs an overline.

Reply:
Title underlining that needs attention is noted in the _raw_ workflow log file with this text: `WARNING: Title underline too short`